assets/src/less/blocks/wp-block-button/layout.less

```diff
@@ -1,29 +1,94 @@
 /**
  * .wp-block-button
  */
-
 .wp-block-button {
 
+	margin-inline-end: @space-1;
+	margin-block-end: @space-1;
+
+	&:last-child {
+		margin-block-end: 0;
+	}
+
 	.wp-block-button__link {
 		.button();
 	}
 
-	&.alignleft {
+	&.has-custom-width {
+
+		max-inline-size: none;
```

Comment:
What does this do? I toggled it on and off but didn't see anything change. I thought maybe it was important if the text is longer than the button width, but that didn't matter either. I see that its included in the WP styles, but I don't know why.